mix.exs

```diff
@@ -1,10 +1,23 @@
 defmodule Silicon.MixProject do
   use Mix.Project
 
+
+  @description """
+    Elixir wrapper for Rust Silicon crate.
+  """
+  @version "./VERSION" |> File.read!() |> String.trim()
```

Comment:
if you wanted to enforce a valid version string
```suggestion
  @version "./VERSION" |> File.read!() |> String.trim() |> Version.parse!() |> Version.to_string()
```

Reply:
Didn't know about this, will add it in - thanks!